src/web/components/Card/components/LabBadge.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { css } from 'emotion';
+import { space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+
+type Props = {
+	badgeForLab: string;
+	brandName?: string;
+	palette: Palette;
+};
+
+// max-height: ${space[12]}px; - Couldn't find a direct replacement in Source
```

Comment:
I think we can remove this comment. As per our chat earlier `space` is used for padding and margins and isn't needed for things like the height of stuff